packages/validator/test/unit/utils/interopConfigs.ts

```diff
@@ -1,154 +1,193 @@
-export const lighthouseHoleskyConfig = {
-  CONFIG_NAME: "holesky",
-  PRESET_BASE: "mainnet",
-  TERMINAL_TOTAL_DIFFICULTY: "0",
-  TERMINAL_BLOCK_HASH: "0x0000000000000000000000000000000000000000000000000000000000000000",
-  TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: "18446744073709551615",
-  SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY: "128",
-  MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: "16384",
-  MIN_GENESIS_TIME: "1695902100",
-  GENESIS_FORK_VERSION: "0x01017000",
-  GENESIS_DELAY: "300",
-  ALTAIR_FORK_VERSION: "0x02017000",
+export const lighthouseHoodiConfig = {
   ALTAIR_FORK_EPOCH: "0",
-  BELLATRIX_FORK_VERSION: "0x03017000",
-  BELLATRIX_FORK_EPOCH: "0",
-  CAPELLA_FORK_VERSION: "0x04017000",
-  CAPELLA_FORK_EPOCH: "256",
-  DENEB_FORK_VERSION: "0x05017000",
-  DENEB_FORK_EPOCH: "29696",
-  SECONDS_PER_SLOT: "12",
-  SECONDS_PER_ETH1_BLOCK: "14",
-  MIN_VALIDATOR_WITHDRAWABILITY_DELAY: "256",
-  SHARD_COMMITTEE_PERIOD: "256",
-  ETH1_FOLLOW_DISTANCE: "2048",
-  SUBNETS_PER_NODE: "2",
-  INACTIVITY_SCORE_BIAS: "4",
-  INACTIVITY_SCORE_RECOVERY_RATE: "16",
-  EJECTION_BALANCE: "28000000000",
-  MIN_PER_EPOCH_CHURN_LIMIT: "4",
-  MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: "8",
-  MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: "128000000000",
-  CHURN_LIMIT_QUOTIENT: "65536",
-  PROPOSER_SCORE_BOOST: "40",
-  DEPOSIT_CHAIN_ID: "17000",
-  DEPOSIT_NETWORK_ID: "17000",
-  DEPOSIT_CONTRACT_ADDRESS: "0x4242424242424242424242424242424242424242",
-  GOSSIP_MAX_SIZE: "10485760",
-  MAX_REQUEST_BLOCKS: "1024",
-  EPOCHS_PER_SUBNET_SUBSCRIPTION: "256",
-  MIN_EPOCHS_FOR_BLOCK_REQUESTS: "33024",
-  MAX_CHUNK_SIZE: "10485760",
-  TTFB_TIMEOUT: "5",
-  RESP_TIMEOUT: "10",
+  ALTAIR_FORK_VERSION: "0x20000910",
   ATTESTATION_PROPAGATION_SLOT_RANGE: "32",
-  MAXIMUM_GOSSIP_CLOCK_DISPARITY_MILLIS: "500",
-  MESSAGE_DOMAIN_INVALID_SNAPPY: "0x00000000",
-  MESSAGE_DOMAIN_VALID_SNAPPY: "0x01000000",
-  ATTESTATION_SUBNET_EXTRA_BITS: "0",
   ATTESTATION_SUBNET_PREFIX_BITS: "6",
-  ATTESTATION_SUBNET_SHUFFLING_PREFIX_BITS: "3",
-  MAX_REQUEST_BLOCKS_DENEB: "128",
-  MAX_REQUEST_BLOB_SIDECARS: "768",
-  MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: "4096",
+  BALANCE_PER_ADDITIONAL_CUSTODY_GROUP: "32000000000",
+  BASE_REWARD_FACTOR: "64",
+  BELLATRIX_FORK_EPOCH: "0",
+  BELLATRIX_FORK_VERSION: "0x30000910",
+  BLOB_SCHEDULE: '[{"EPOCH": "52480", "MAX_BLOBS_PER_BLOCK": "15"}, {"EPOCH": "54016", "MAX_BLOBS_PER_BLOCK": "21"}]',
```

Comment:
what's the reason that this is a stringifed array? client should be returning this as an json array